crates/cli/src/main.rs

```diff
@@ -49,7 +49,7 @@ fn main() -> io::Result<()> {
                     &matches,
                     &subcommands,
                     BuildConfig::BuildAndRunIfNoErrors,
-                    Triple::host(),
+                    Triple::host().try_into().unwrap(),
```

Comment:
Hm, can we be sure it's safe to unwrap this? Seems like we should try to give a nicer error if the user is building on an unsupported platform 😄

Reply:
It totally isn't safe though the behavior is no different than before. It would have just failed deeper in the compiler when convert to the old version of roc_target. let me think of a nicer way to handle this, it happens in a number of places.